src/components/BookingForm.vue

```diff
@@ -169,6 +182,11 @@ export default {
       this.messageType = "error";
       this.isMessageShownOnBooking = true;
     },
+    async bookADesk() {
```

Comment:
The code for book a desk has some redundant calls.
bookADesk calls:
1. submitBooking, which then:
    - Calls post on /bookings
    - If there are no errors, it calls fetchAvailabilities and then calls fetchBookings
    - If there is an error, calls displayErrorMessage
2. fetchAvailabilities (already done)
3. fetchBookingsList

From this, I think:
1. We should somehow reconcile both submitBooking and bookADesk, since they both have redundant calls
2. Look into if fetchBookings is necessary. I forget exactly why it was used, but I don't know if it's necessary anymore
    - If fetchBookings is no longer necessary, please remove it. Also rename fetchBookingsList to fetchBookings. When naming, you generally want to avoid including type information in the function (such as Array or List). The important part is that the bookings are being fetched, not that they come in a list or not.
    - If fetchBookings is necessary, I think we should rename fetchBookings and fetchBookingsList in a way that describes in what feature they are being used. Right now, they essentially have the same name which makes it very confusing to understand the existence of either function

Reply:
Ok, so I've figured out something about `fetchBookings`. We have a partially implemented UI that handles "Upcoming Bookings" (ie: what bookings the user have that are upcoming". This is displayed in the Dashboard.vue file.

I think the goal was that Dashboard.vue becomes the new main vue instead of BookingForm.vue.

I think there are a couple of things we should discuss in person friday:
1. How to handle both the upcoming bookings and the Who's In Vue.
2. How to refactor Booking Form so that it becomes less of a god class
3. How to use the concept of a store (which is what `fetchBookings` is using). I think this will greatly simplify the code once we properly use the concept.

For now: remove `fetchBookings`, we will reimplement the logic later, after we've done a refactor and when we start to reimplement the "Upcoming Bookings" logic

Reply:
*also to note: points 1-3 are not required for your PR. We will work on this on a separate PR to make sure each PR does not grow too much and becomes too hard to understand